modules/registry-configuring-storage-baremetal.adoc

```diff
@@ -16,9 +16,13 @@ registry to use storage.
 
 * Cluster administrator permissions.
 * A cluster on bare metal.
-* Provisioned persistent storage
-for your cluster, such as Red Hat OpenShift Container Storage. To deploy a private image registry, your storage must provide
-ReadWriteMany access mode.
+* Persistent storage provisioned for your cluster.
++
+[IMPORTANT]
+====
+{product-title} supports `ReadWriteOnce` access mode for image registry storage when you have only one replica. To deploy a private image registry, or to configure registry storage for high availability with two or more replicas, you must use a different storage back end that supports `ReadWriteMany`, such as object storage.
+====
```

Comment:
I don't know if we support object storage in baremetal cluster?
@wzheng1 Did we test object storage as backend storage for image registry?

Reply:
Image registry is recommended to use object storage (S3, GCS, Swift, etc.), but I don't know what object storage can be used in a baremetal cluster?

Reply:
I suggest to remove these statement, since they are no related to Baremetal and vSphere configuration: to configure registry storage for high availability with two or more replicas, you must use a different storage back end that supports `ReadWriteMany`, such as object storage.

Reply:
^ This statement was included as a rewording of this statement by @dmage:

> Image registry storage requires ReadWriteMany only for HA configuration (2+ replicas). Since 4.4 there is an option to use RWO if you have only one replica.

So perhaps we could leave the proposed statement as is for vSphere, and drop "such as object storage" for bare metal?

Reply:
One more thing is about "To deploy a private image registry", maybe it should be To deploy an image registry which supports HA, ReadWriteMany is needed.

Reply:
Resolved